src/index.js

```diff
@@ -0,0 +1,213 @@
+    tempSetting = 5;
+  } else if (adjustedTemp > 79) {
+    tempSetting = 4;
+  } else if (adjustedTemp > 65) {
+    tempSetting = 3;
+  } else if (adjustedTemp > 39) {
+    tempSetting = 2;
+  } else if (adjustedTemp > 14) {
+    tempSetting = 1;
+  } else if (adjustedTemp <= 14) {
+    tempSetting = 0;
+  }
+
+  currentTemp = adjustedTemp;
+
+  showField.innerHTML = HILLS_ARE_ALIVE[tempSetting];
+  showYou.innerHTML = HOW_YA_FEELIN[tempSetting];
+  shownTemp.innerHTML = adjustedTemp;
+  document.getElementById('temp_result').style.color =
+    REPO_RAINBOW[tempSetting];
+};
+
+// TEMPERATURE
+
+let shownTemp = document.getElementById('temp_result');
+
+shownTemp.innerHTML = currentTemp;
+
+const handleAdd = () => {
+  currentTemp++;
+  shownTemp.innerHTML = currentTemp;
+
+  updateTempies(currentTemp);
+};
+
+const handleSub = () => {
+  currentTemp--;
+  shownTemp.innerHTML = currentTemp;
+
+  updateTempies(currentTemp);
+};
+
+// API CALLS
+
+const findWeather = (query) => {
+  let locationData, weatherData;
+  let modifiedLocData, modifiedWeathData;
+
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: currentCity,
+        format: 'json',
+      },
+    })
+    .then((response) => {
+      locationData = response.data;
+
+      const modifyLocation = (locationData) => {
+        let latlon = [];
+        let focusLocation = locationData[0];
+
+        let focusLat = Number(focusLocation['lat']);
+        let adjustLat = focusLat.toFixed(2);
+        latlon.push(adjustLat);
+
+        let focusLon = Number(focusLocation['lon']);
+        let adjustLon = focusLon.toFixed(2);
+        latlon.push(adjustLon);
+
+        return latlon;
+      };
+
+      modifiedLocData = modifyLocation(locationData);
+
+      axios
+        .get('http://127.0.0.1:5000/weather', {
+          params: {
+            lat: modifiedLocData[0],
+            lon: modifiedLocData[1],
+            format: 'json',
+          },
+        })
+        .then((response) => {
+          weatherData = response.data;
+
+          const modifyWeather = (weatherData) => {
+            let relevantData = [weatherData['main'], weatherData['weather']];
+            let defTempCond = [];
+
+            let unmodifiedTemp = relevantData[0]['temp'];
+            let tempF = ((unmodifiedTemp - 273.15) * 9) / 5 + 32;
+            let modifiedTemp = tempF.toFixed(0);
+            defTempCond.push(modifiedTemp);
+
+            let condID = Number(relevantData[1][0]['id']);
+            let idToCond = 'sunny';
+
+            if (condID > 800) {
+              idToCond = '⛅';
+            } else if (condID === 800) {
+              idToCond = '✨';
+            } else if (condID > 700) {
+              idToCond = '⛅';
+            } else if (condID > 599) {
+              idToCond = '☃️';
+            } else {
+              idToCond = '🌈';
+            }
+
+            defTempCond.push(idToCond);
+            return defTempCond;
+          };
+
```

Comment:
Consider moving this outside of the function to separate concerns but great work with this approach 🏁